packages/viewer/src/lib/types/frameCamera.ts

```diff
@@ -60,25 +79,24 @@ export function toOrthographic(
     up: data.up,
     lookAt: data.lookAt,
     fovHeight:
-      2 * Vector3.magnitude(viewVector) * Math.tan(Angle.toRadians(22.5)),
+      2 *
+      Vector3.magnitude(viewVector) *
+      Math.tan(Angle.toRadians(data.fovY / 2.0)),
   };
 }
 
 export function toPerspective(
   data: OrthographicFrameCamera
 ): PerspectiveFrameCamera {
-  const expectedMagnitude =
-    data.fovHeight / (2 * Math.tan(Angle.toRadians(22.5)));
-  const receivedMagnitude = Vector3.magnitude(data.viewVector);
-  const magnitudeScale = expectedMagnitude / receivedMagnitude;
+  const fovY = Angle.toDegrees(
+    Math.atan(data.fovHeight / (2 * Vector3.magnitude(data.viewVector))) * 2
```

Comment:
I think you want to leave this the way you had it before (`fovY==45`, solve for view-vector length) -- is there a reason you changed it?

Maybe `45.0` should be passed in as a parameter...

Reply:
Yeah, based on my testing the `magnitudeScale` was always 1 (or extremely close - like 1.000034), so it wasn't clear if we needed to keep that around since we maintain the magnitude of the `viewVector` when we convert to orthographic. As for passing in the `fovY`, it seems like we may run into problems if we start from an orthographic camera (since that fovY value would be unknown at that point). Would there be a way to solve for both the magnitude of the `viewVector` and the `fovY` given a starting orthographic camera with our current representation?

Reply:
I see what you're saying... There might be something clever you could do here but I can't see it at the moment.

In general, you would not expect `fovY` to change often, but a client may have a specific idea of what it ought to be (e.g. at PLC, the standard was `56`).  That's the level of flexibility we should be supporting -- once set, all individual cameras are very likely going to use the same setting.  I don't know how the SDK handles the concept of "application defaults" but this feels like one of those situations.

Reply:
Yeah, I think that makes sense. For the purposes of this addition, I may lean toward not including a configuration option or property (since it won't do anything until we wire up the backend). For now I'll add an optional `fovY` argument for that `toPerspective()` method that just defaults to `45` with a TODO to revisit once we allow changing that value